pkg/status/status_server.go

```diff
@@ -295,7 +294,7 @@ func (s *Server) bpfLogLevel(w http.ResponseWriter, r *http.Request) {
 		}
 		key := uint32(0)
 		levelPtr := uint32(level)
-		if err := s.bpfWorkloadObj.SockConn.BpfLogLevel.Update(&key, &levelPtr, ebpf.UpdateAny); err != nil {
+		if err := s.bpfLogLevelMap.Update(&key, &levelPtr, ebpf.UpdateAny); err != nil {
```

Comment:
Safer way is to check whether bpfLogLevelMap is nil. Kmesh maybe started only with `bypass`

Reply:
Done.